src/sql/platform/accounts/browser/accountPicker.ts

```diff
@@ -157,7 +157,7 @@ export class AccountPicker extends Disposable {
 		this._onAccountSelectionChangeEvent.fire(account);
 	}
 
-	private renderLabel(container: HTMLElement): IDisposable {
+	private renderLabel(container: HTMLElement): IDisposable | null {
```

Comment:
>null [](start = 60, length = 4)

Why use null here? And looking at this why isn't this just void? We never actually return an IDisposable

Reply:
Its an interface defined by vscode